### DISCUSSION TITLE
Can't import child_process

### DISCUSSION THREAD

**Discussion by taxuexunshang** [webpack/webpack#18142] [Q&A]
Hello, I want to use child_process to invoke an exe file in a React + Electron Project. But some error happend when I import child_process module.
The first is  `Module not found: Error: Can't resolve 'child_process'`
I solve it by add 
`"browser": {
    "child_process": false 
 }` to `package.json` ,It package  successfully.
But when I click a button which use child_process  invoke an external exe.  It will throw a second error `spawn() is not a function`.
I have tried
- [ x ]  change `import { spawn } from 'child_process'` to `const { spawn } = require('child_process')`

The project sorece code in [electron-react-boilerplate](https://github.com/electron-react-boilerplate/electron-react-boilerplate), I just add a button to test child_process spawn() invoke an exe.
Please help me, thanks~

**Comment by alexander-akait** [comment]
You can't use `child_process` in a browser

**Reply by taxuexunshang** [reply]
Thanks for your help, After I packaged it as exe, the error still exists. Please tell me the reason. Many thanks.

**Reply by alexander-akait** [reply]
Please ask this in https://github.com/electron-react-boilerplate/electron-react-boilerplate repo

**Reply by taxuexunshang** [reply]
OK, thanks for your help!
